code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -189,6 +189,10 @@
 	///The iconstate for the xeno on the minimap
 	var/minimap_icon = "xeno"
 
+
+/datum/xeno_caste/proc/buff_xeno_damage()
```

Comment:
Autodoc
Should also have update or apply in the name as this doesnt actually set it